engines/director/util.h

```diff
@@ -101,6 +101,25 @@ Common::String utf8ToPrintable(const Common::String &str);
 
 Common::String decodePlatformEncoding(Common::String input);
 
+inline byte lerpByte(byte a, byte b, int alpha, int span) {
```

Comment:
what does `lerp` mean?

Reply:
Short for linear interpolation - https://en.wikipedia.org/wiki/Linear_interpolation#History_and_applications